src/index.js

```diff
@@ -163,6 +165,203 @@ class Radar {
       }
     );
   }
+
+  static searchPlaces(latitude, longitude, radius, chains, categories, groups, limit, callback) {
+    const publishableKey = Cookie.getCookie(Cookie.PUBLISHABLE_KEY);
+
+    if (!publishableKey) {
+      if (callback) callback(STATUS.ERROR_PUBLISHABLE_KEY);
```

Comment:
nit: let's keep the braces on if statements for consistency. We should probably add some basic linting at some point.